df.world-site.xml

```diff
@@ -431,14 +431,16 @@
             <static-array count='51' pointer-type='site_realization_crossroads'/>
         </static-array>
         <static-array name='river_map' count='51'>
-            <pointer>
-                <static-array name='start' type-name='int32_t' count='48'/>
-                <static-array name='end'   type-name='int32_t' count='48'/>
-                <int32_t name="unk_180"/>
-                <int32_t name="unk_184"/>
-                <int32_t name="start_array_limits"/>
-                <int32_t name="end_array_limits"/>
-            </pointer>
+            <static-array count='51'> 
```

Comment:
Also river_map is a double dimension array of pointers to the proposal unknown structure.
Previously was one array of void pointers. I only changed the type of the pointer.

[51]x[51]x4 = 10404, so there's no size difference (4 = sizeof pointer)